### DISCUSSION TITLE
Add empty paragraph at the end of the editor

### DISCUSSION THREAD

**Discussion by zgrybus** [Milkdown/milkdown#952] [Q&A] [answered]
Good morning,
I have one simple question. What should I do so that after adding any field, an empty paragraph always appears at the end?
Exactly the same as it is done at https://milkdown.dev/playground
I ask because when I add a table, I can't add any text because there is no paragraph there.

**Comment by Saul-Mirone** [comment, marked as answer]
https://milkdown.dev/docs/api/plugin-trailing
